mantidproject/mantid
# Script Editor Zoom

**Description of work.**
This pull request makes it possible to use the keyboard shortcuts Ctrl++ and Ctrl+- to zoom in and out of the script editor. This feature is now present in both MantidPlot and Mantid Workbench.

**To test:**
1. Open Workbench, click on the script editor area, check that Ctrl++ zooms in and Ctrl+- zooms out (both the keys next to backspace and the num pad keys).
2. Repeat for MantidPlot.

Fixes #25800 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
